### DISCUSSION TITLE
Make the install process/button more self evident

### DISCUSSION THREAD

**Discussion by Mozoloa** [rustdesk/rustdesk#10074] [General]
Every time I tell a client to get rustdesk, they invariably fail to install it and just launch it as is, with all the glitches that come with it. And when I tell them to install it they don't understand because the install button in the sidebar is pushed out of the window lower edge by a confusing wall of text, forcing people to resize the window to even access such a basic functionality.

This has happened hundreds of times (not kidding), and could be easily fixed by moving it in a more obvious spot, or shortening the wall of text, or placing the buttons first before the text, or preferably, **everything at the same time.**
![image](https://github.com/user-attachments/assets/d2c1bb70-b4aa-4dbe-bbe0-a11570733194)

**Comment by rustdesk** [comment]
Could you help to shorten this translation of warning text? I guess it is not hard for a native speaker. https://github.com/rustdesk/rustdesk/tree/master/src/lang

**Comment by Mozoloa** [comment]
I was wondering if that wasn't just a translation issue, indeed the french text is way too long, lemme make a pr
